src/Controllers/LoggerController.js

```diff
@@ -4,6 +4,8 @@ import AdaptableController from './AdaptableController';
 import { LoggerAdapter } from '../Adapters/Logger/LoggerAdapter';
 
 const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000;
+const logStringTruncateLength = 1000;
```

Comment:
Nope not adjustable :) I know we're not supposed to use upper case strings as var names but that's more readable. Can you make it LOG_STRING_TRUNCATE_LENGTH?